lib/screens/app/guardians/im_guardian_for_tab.dart

```diff
@@ -220,95 +220,99 @@ class ImGuardianForTab extends StatelessWidget {
   }
 
   void showStopRecoveryConfirmationDialog(MemberModel user, BuildContext context) {
-    showDialog(
-        context: context,
-        child: AlertDialog(
-            content: Text("Are you sure you want to stop key recovery process", style: TextStyle(color: Colors.black)),
-            actions: [
-              FlatButton(
-                child: const Text('Cancel'),
-                onPressed: () {
-                  Navigator.pop(context);
-                },
-              ),
-              FlatButton(
-                child: Text("Stop Key Recovery"),
-                onPressed: () {
-                  FirebaseDatabaseService().stopRecoveryForUser(userId: user.account);
-                  Navigator.pop(context);
-                  Navigator.pop(context);
-                },
-              )
-            ]));
+    // TODO(gguij002): Trouble migrating to flutter 2, Fix in v2
```

Comment:
This is because FlatButton is deprecated i guess too

Reply:
yes